Cargo.toml

```diff
@@ -30,6 +31,7 @@ debug-assertions = false
 
 [profile.dev]
 opt-level = 0
+panic = 'abort'
```

Comment:
Is there a benefit to turning it to `abort` here?

Reply:
Not to my knowledge, but it feels like a safer default imo